newspaper/network.py

```diff
@@ -68,7 +68,13 @@ def get_html_2XX_only(url, config=None, response=None):
         # fail if HTTP sends a non 2XX response
         response.raise_for_status()
 
-    return html
+    # Check to see if the Content-Type header exists
+    mime_type = None
+    if 'content-type' in response.headers:
+        mime_type = response.headers['content-type']
```

Comment:
This isn't necessarily correct. Content-Type is often made up of a MIME type combined with a charset like the one below:
`text/html; charset=utf-8`

Reply:
Based on [`Content-Type` docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) this seems like the right solution. Suggestions for a better header to take?

Reply:
One option would be to parse out stuff to `;` and take only the first part.

Reply:
I think the header is  the correct one to take, but if we're setting mime_type and not content_type then I'd do something like `mime_type = response.headers.get('content-type').split(';')[0]`

Reply:
OK, will do, that makes sense. :)